application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -90,9 +91,17 @@ private void handleHelpThreadCreated(ThreadChannel threadChannel) {
         SERVICE.schedule(createMessages, 5, TimeUnit.SECONDS);
     }
 
+    private RestAction<Message> createAIResponse(ThreadChannel threadChannel) {
+        RestAction<Message> originalQuestion =
+                threadChannel.retrieveMessageById(threadChannel.getIdLong());
```

Comment:
y u used the thread channel id to get the first message? is that valid?

Reply:
It works if that's what you mean. Calling threadChannel.getParentMessage causes an error:
```Parent of this thread is not a MessageChannel. Parent: net.dv8tion.jda.internal.entities.ForumChannelImpl@57f68dd7```

I think I found some documentation somewhere that states that using the thread channels id will return the first message. Just tried to look for it and couldn't find it.

Is there another method I should be using?